kolibri/core/assets/src/views/language-switcher/modal.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+
+  <div>
+    <core-modal
+      :title="$tr('changeLanguageModalHeader')"
+      @enter="setLang"
+      @cancel="closeModal"
+    >
+      <p>{{ $tr('changeLanguageSubHeader') }}</p>
+      <k-radio-button
+        v-for="language in languageOptions"
+        :key="language.id"
+        :radiovalue="language.id"
+        :label="language.lang_name"
+        v-model="selectedLanguage"
+      />
+      <div class="footer">
+        <k-button :text="$tr('cancelButtonText')" :raised="false" @click="closeModal"/>
+        <k-button :text="$tr('confirmButtonText')" :primary="true" @click="setLang"/>
+      </div>
+    </core-modal>
+  </div>
+
+</template>
+
+
+<script>
+
+  import coreModal from 'kolibri.coreVue.components.coreModal';
+  import kButton from 'kolibri.coreVue.components.kButton';
+  import kRadioButton from 'kolibri.coreVue.components.kRadioButton';
+  import languageSwitcherMixin from 'kolibri.coreVue.mixins.languageSwitcherMixin';
+  import { currentLanguage } from 'kolibri.utils.i18n';
+
+  export default {
+    name: 'languageSwitcherModal',
+    components: { coreModal, kButton, kRadioButton },
+    mixins: [languageSwitcherMixin],
+    $trs: {
+      changeLanguageModalHeader: 'Change language',
```

Comment:
I assume the strings are unchanged? We will need to reupload strings and run TM after this, as the ids will have changed.